src/test/java/recipeditor/command/FindCommandTest.java

```diff
@@ -54,9 +63,12 @@ void incorrectHelpCommand_invalidInput_syntaxForFindCommand() {
                 + "-t: Recipe Title\n"
                 + "-i: Ingredient name";
         assertEquals(expected, commandExecutedResult.getMessage());
+    }
+
+    @AfterAll
+    static void end() {
         RecipeList.deleteRecipeFromTitle("Example title");
         Storage.deleteRecipeFile("Example title");
-        Storage.rewriteRecipeListToFile(Storage.ALL_RECIPES_FILE_PATH);
```

Comment:
I believe this is a fix? Since the original line will destroy the existing saved recipes, which is a very dangerous bug.